src/components/cc-token-api-creation-form/cc-token-api-creation-form.stories.js

```diff
@@ -0,0 +1,751 @@
+import { DateFormatter } from '../../lib/date/date-formatter.js';
+import { shiftDateField } from '../../lib/date/date-utils.js';
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import { CcTokenApiCreationForm } from './cc-token-api-creation-form.js';
+
+const dateFormatter = new DateFormatter('datetime-short', 'local');
+
+/**
+ * @param {CcTokenApiCreationForm} component
+ * @param {'configuration-form'|'validation-form'} formName
+ * @returns {HTMLFormElement}
+ */
+const getForm = function (component, formName) {
+  return component.shadowRoot.querySelector(`form[name=${formName}]`);
+};
```

Comment:
Should the component itself expose both forms through getters instead?

Reply:
note: I think what you did is OK for now. We'll have to think about what we want for general component based e2e tests. Do we want "page objects" but for components? If yes, is it part of the component API? Lots of interesting discussions to come.